tools/config/__init__.py

```diff
@@ -696,7 +748,8 @@ def _assign_new_offset(rom_start, start, new_offset, region_name):
                 "Can not place % region inside previous region" % region_name)
         return newstart
 
-    def _generate_bootloader_build(self, rom_start, rom_size):
+    def _generate_bootloader_build(self, active_rom_regions):
+        rom_start, rom_size = next(iter(active_rom_regions.values()))
```

Comment:
In tandem with the above:
```suggestion
        rom_start, rom_size = ROM or next(iter(other_rom_memories.values()))
```